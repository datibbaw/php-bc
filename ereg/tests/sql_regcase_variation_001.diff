006+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
007+ string(0) ""
006- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
007- string(1) "0"
010+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
011+ string(0) ""
010- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
011- string(1) "1"
014+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
015+ string(0) ""
014- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
015- string(5) "12345"
018+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
019+ string(0) ""
018- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
019- string(5) "-2345"
022+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
023+ string(0) ""
022- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
023- string(4) "10.5"
026+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
027+ string(0) ""
026- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
027- string(5) "-10.5"
030+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
031+ string(0) ""
030- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
031- string(12) "101234567000"
034+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
035+ string(0) ""
034- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
035- string(16) "1.07654321[Ee]-9"
038+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
039+ string(0) ""
040+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)
041+ 
042+ Arg value Array 
043+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
044+ Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
045+ string(0) ""
046+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)
047+ 
048+ Arg value Array 
049+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
050+ Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
051+ string(0) ""
052+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)
053+ 
054+ Arg value Array 
055+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
056+ Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
057+ string(0) ""
058+ Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)
059+ 
060+ Arg value Array 
061+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
062+ Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
063+ string(0) ""
038- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
039- string(3) "0.5"
043- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
044- Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
045- NULL
046- Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)
048- Arg value Array 
049- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
050- Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
051- NULL
067+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
068+ Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
069+ string(0) ""
070+ 
071+ Arg value  
072+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
073+ string(0) ""
074+ 
075+ Arg value  
076+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
077+ string(0) ""
078+ 
079+ Arg value 1 
080+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
081+ string(0) ""
082+ 
083+ Arg value  
084+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
085+ string(0) ""
086+ 
087+ Arg value 1 
088+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
089+ string(0) ""
090+ 
091+ Arg value  
092+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
093+ string(0) ""
094+ 
095+ Arg value  
096+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
097+ string(0) ""
098+ 
099+ Arg value  
100+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
101+ string(0) ""
102+ Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)
103+ 
104+ Arg value  
105+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
106+ Error: 2 - strlen() expects parameter 1 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
107+ string(0) ""
108+ 
109+ Arg value  
110+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
111+ string(0) ""
113+ Arg value  
114+ Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
115+ string(0) ""
116+ Done
052- Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)
053- 
054- Arg value Array 
055- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
056- Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
057- NULL
058- Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)
059- 
060- Arg value Array 
061- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
062- Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
063- NULL
064- Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)
065- 
066- Arg value Array 
067- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
068- Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
069- NULL
070- 
071- Arg value  
072- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
073- string(0) ""
074- 
075- Arg value  
076- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
077- string(0) ""
078- 
079- Arg value 1 
080- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
081- string(1) "1"
082- 
083- Arg value  
084- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
085- string(0) ""
086- 
087- Arg value 1 
088- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
089- string(1) "1"
090- 
091- Arg value  
092- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
093- string(0) ""
094- 
095- Arg value  
096- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
097- string(0) ""
098- 
099- Arg value  
100- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
101- string(0) ""
102- Error: 4096 - Object of class stdClass could not be converted to string, %s(71)
103- 
104- Arg value  
105- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
106- Error: 2 - sql_regcase() expects parameter 1 to be string, object given, %s(72)
107- NULL
108- 
109- Arg value  
110- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
111- string(0) ""
112- 
113- Arg value  
114- Error: 8192 - Function sql_regcase() is deprecated, %s(72)
115- string(0) ""
116- Done