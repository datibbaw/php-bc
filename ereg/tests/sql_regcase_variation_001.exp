*** Testing sql_regcase() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(62)
Error: 8 - Undefined variable: unset_var, %s(65)

Arg value 0 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "0"

Arg value 1 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "1"

Arg value 12345 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(5) "12345"

Arg value -2345 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(5) "-2345"

Arg value 10.5 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(4) "10.5"

Arg value -10.5 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(5) "-10.5"

Arg value 101234567000 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(12) "101234567000"

Arg value 1.07654321E-9 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(16) "1.07654321[Ee]-9"

Arg value 0.5 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(3) "0.5"
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value 1 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "1"

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value 1 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "1"

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""
Error: 4096 - Object of class stdClass could not be converted to string, %s(71)

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, object given, %s(72)
NULL

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""
Done