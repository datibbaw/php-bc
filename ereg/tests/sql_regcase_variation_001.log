
---- EXPECTED OUTPUT
*** Testing sql_regcase() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(62)
Error: 8 - Undefined variable: unset_var, %s(65)

Arg value 0 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "0"

Arg value 1 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "1"

Arg value 12345 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(5) "12345"

Arg value -2345 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(5) "-2345"

Arg value 10.5 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(4) "10.5"

Arg value -10.5 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(5) "-10.5"

Arg value 101234567000 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(12) "101234567000"

Arg value 1.07654321E-9 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(16) "1.07654321[Ee]-9"

Arg value 0.5 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(3) "0.5"
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL
Error: 8 - Array to string conversion, %ssql_regcase_variation_001.php(%d)

Arg value Array 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, array given, %s(72)
NULL

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value 1 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "1"

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value 1 
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(1) "1"

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""
Error: 4096 - Object of class stdClass could not be converted to string, %s(71)

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
Error: 2 - sql_regcase() expects parameter 1 to be string, object given, %s(72)
NULL

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""

Arg value  
Error: 8192 - Function sql_regcase() is deprecated, %s(72)
string(0) ""
Done
---- ACTUAL OUTPUT
*** Testing sql_regcase() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(62)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(65)

Arg value 0 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 12345 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value -2345 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 10.5 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value -10.5 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 101234567000 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1.07654321E-9 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 0.5 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""
Done
---- FAILED
