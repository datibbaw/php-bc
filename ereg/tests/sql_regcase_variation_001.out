*** Testing sql_regcase() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(62)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(65)

Arg value 0 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 12345 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value -2345 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 10.5 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value -10.5 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 101234567000 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1.07654321E-9 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 0.5 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value Array 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, array given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value 1 
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/sql_regcase_variation_001.php(71)

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
Error: 2 - strlen() expects parameter 1 to be string, object given, /Users/tjerk/Documents/php-bc/ereg/ereg.php(211)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""

Arg value  
Error: 16384 - Function sql_regcase() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(209)
string(0) ""
Done